src/datepicker/datepicker.js

```diff
@@ -150,7 +151,9 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.position'])
       }
 
       // Configuration parameters
-      var mode = 0, selected = new Date(), showWeeks = datepickerConfig.showWeeks;
+      var mode = 0;
+      var selected = scope.focusedDate = new Date();
```

Comment:
The problem with this, is that it has an isolated scope, so I cannot watch for changes outside of the directive. I have to expose it so that it can be watched.